src/Uno.UI/UI/Xaml/Controls/TimePicker/TimePickerSelector.iOS.cs

```diff
@@ -100,11 +125,23 @@ private void SetTime(TimeSpan newTime)
 				return;
 			}
 
-			var localeID = newClockIdentifier == ClockIdentifiers.TwelveHour 
-				? "en" 
+			var localeID = newClockIdentifier == ClockIdentifiers.TwelveHour
+				? "en"
 				: "fr";
```

Comment:
`en` & `fr` are the one 2 possible values here?

Reply:
My understanding is that UWP has ClockIdentifier which is 24h format or 12h format, but in iOS it is based on the Locale, so thisi s the way that was used to translate the UWP enum into the iOS native picker. This is my understanding at least, we briefly looked  at it because we also thought it was odd :)